Sources/Testing/Expectations/Expectation+Macro.swift

```diff
@@ -620,87 +551,20 @@ public macro require<R>(
-/// then suspended (as with `await`) and waits for the child process to
-/// terminate. `expression` is not called in the parent process.
-///
-/// Meanwhile, in the child process, `expression` is called directly. To ensure
-/// a clean environment for execution, it is not called within the context of
-/// the original test. If `expression` does not terminate the child process, the
-/// process is terminated automatically as if the main function of the child
-/// process were allowed to return naturally. If an error is thrown from
-/// `expression`, it is handed as if the error were thrown from `main()` and the
-/// process is terminated.
-///
-/// Once the child process terminates, the parent process resumes and compares
-/// its exit status against `expectedExitCondition`. If they match, the exit
-/// test has passed; otherwise, it has failed and an issue is recorded.
-///
-/// ## Child process output
-///
-/// By default, the child process is configured without a standard output or
-/// standard error stream. If your test needs to review the content of either of
-/// these streams, you can pass its key path in the `observedValues` argument:
-///
-/// ```swift
-/// let result = try await #require(
-///   exitsWith: .failure,
-///   observing: [\.standardOutputContent]
-/// ) {
-///   print("Goodbye, world!")
-///   fatalError()
-/// }
-/// #expect(result.standardOutputContent.contains(UInt8(ascii: "G")))
-/// ```
-///
-/// - Note: The content of the standard output and standard error streams may
-///   contain any arbitrary sequence of bytes, including sequences that are not
-///   valid UTF-8 and cannot be decoded by [`String.init(cString:)`](https://developer.apple.com/documentation/swift/string/init(cstring:)-6kr8s).
-///   These streams are globally accessible within the child process, and any
-///   code running in an exit test may write to it including the operating
-///   system and any third-party dependencies you have declared in your package.
-///
-/// The actual exit condition of the child process is always reported by the
-/// testing library even if you do not specify it in `observedValues`.
-///
-/// ## Runtime constraints
-///
-/// Exit tests cannot capture any state originating in the parent process or
-/// from the enclosing lexical context. For example, the following exit test
-/// will fail to compile because it captures an argument to the enclosing
-/// parameterized test:
-///
-/// ```swift
-/// @Test(arguments: 100 ..< 200)
-/// func sellIceCreamCones(count: Int) async throws {
-///   try await #require(exitsWith: .failure) {
-///     precondition(
-///       count < 10, // ERROR: A C function pointer cannot be formed from a
-///                   // closure that captures context
-///       "Too many ice cream cones"
-///     )
-///   }
+/// @Metadata {
+///   @Available(Swift, introduced: 6.2)
 /// }
-/// ```
-///
-/// An exit test cannot run within another exit test.
```

Comment:
I see most of the content removed here has been adapted in the `<doc:exit-testing>` article, but I don't see a reference to this sentence

> An exit test cannot run within another exit test.

Reply:
Might have gotten elided during editing. It isn't worth its own section, but maybe put it as a `- Note` callout somewhere reasonable?